libs/dh/shared/feature-authorization/src/lib/permission.service.ts

```diff
@@ -36,24 +41,26 @@ export class PermissionService {
       return of(true);
     }
 
-    const account = this.authService.instance.getActiveAccount();
+    const accounts = this.authService.instance.getAllAccounts();
 
-    // If MSAL returns no accounts or the claims are missing,
-    // the service default to no access.
-    if (!account) {
+    // we expect one and only one account for now.
+    if (accounts.length != 1) {
       return of(false);
     }
 
-    const activeScope = this.scopeService.getActiveScope();
+    const account = accounts[0];
 
     const tokenRequest: SilentRequest = {
-      scopes: [activeScope],
+      scopes: [this.config.backendId ?? this.config.clientId],
```

Comment:
suggestion: maybe one more place where an OR `||` should be preferred?

```suggestion
      scopes: [this.config.backendId || this.config.clientId],
```